esy.json

```diff
@@ -5,19 +5,20 @@
   "repository": "https://github.com/briskml/brisk.git",
   "license": "MIT",
   "scripts": {
+    "fmt": "dune build @fmt --auto-promote",
     "clean": "dune clean",
     "build:core": "esy build -p @brisk/core",
-    "build:mac": "esy build -p @brisk/renderer-macos",
+    "build:mac": "esy build -p @brisk/macos",
```

Comment:
shouldn't we keep this consistent and keep esy scripts local to relevant directories? I mean it depends on your workflow but I usually do core/reconciler stuff inside `core` dir. I'd expect to build the tester inside the tester dir etc. Does it make sense?

Reply:
You mean keep those script commands in ‘core/esy.json’? It does make sense, but esy doesn’t support it. 😁

Reply:
Ah, you mean rename the directory as well? I thought of that, but then it will be all over the place, and I wanted to keep renderers close to each other.

Reply:
Finally understood what you mean. You can work on `core` inside core directory, but I usually run the full project build to make sure nothing's broken.

This scripts are mostly for CI convenience, because I use `esy run:mac` which will run the app in `exec` environment, thus, making sure we got everything we need installed in there.

Reply:
makes sense!